Do a sample Code review

sabetech opened the pull request.

sabetech committed:
copy from files from previous microverse trials

sabetech committed:
Modify index.html with more content

sabetech commented on index.html:
```diff
@@ -1,28 +1,101 @@
-<html lang="en">
+<html>
+    <head>
+        <link href="css/style.css" rel="stylesheet" >
```
Meta tags should go here. They help give more description to your site

Wes-Isaac replied:
- [x] t's better to be more polite you can say _Kindly use this..._ _or it's better to..._

sabetech replied:
Thanks for the feedback

Kingstalux replied:
Add your updated comments here.

sabetech replied:
Kindly use Meta tags in the head section of your website. They help to give more description to your site

sabetech commented on index.html:
```diff
@@ -1,28 +1,101 @@
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
+        <div class="history" id="tribute-info">
```
Instead of a div, you could a section here.

Wes-Isaac replied:
- [x] Good job phrasing your comment. but you have to also provide a reason why you are requesting this particular change and also it's better to be more polite you can say _Kindly use this..._

sabetech replied:
Feedback appreciated

Kingstalux replied:
Add your updated comments here.

sabetech replied:
Kindly use the section tag instead of div. This is semantically more accurate than using a div.

sabetech reviewed (commented):
Looks like the you pasted the html code twice. Check and make that fix

Wes-Isaac reviewed (changes requested):
Hi @sabetech , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sabetech committed:
Post code review changes
- change div to section for the history section
- removed the duplicate html
- included the metatags

Kingstalux reviewed (changes requested):
Hi @sabetech , 

Great work so far 👏 
But you have not applied the requested changes. You are to edit your review comments following the previous reviewer's suggestions. So just go under his comments and add your edited review comments and you will be good to go 👍 


 ## Required Changes ♻️
_Check the comments under your review comments._
- [ ] (OPTIONAL) It would be great if you could add checkboxes to your comments so that the student can tick them when the issue has been resolved 💯 
- [ ] (OPTIONAL) It is nice to add emojis to your suggestions to make them more friendly 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Wes-Isaac reviewed (approved):
Hi @sabetech , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
